jetcd-core/src/test/java/io/etcd/jetcd/ClientBuilderTest.java

```diff
@@ -53,4 +58,45 @@ public void testMaxInboundMessageSize() throws URISyntaxException {
 
     assertThat(channelBuilder).hasFieldOrPropertyWithValue("maxInboundMessageSize", value);
   }
+
+  @Test
+  public void testNamespace() {
+    class TestCase {
+      public ByteSequence namespace;
+      public ByteSequence expectedNamespace;
+
+      public TestCase(ByteSequence namespace, ByteSequence expectedNamespace) {
+        this.namespace = namespace;
+        this.expectedNamespace = expectedNamespace;
+      }
+    }
+    List<TestCase> testCases = Arrays.asList(
+        // namespace setting and expected namespace
+        new TestCase(ByteSequence.EMPTY, ByteSequence.EMPTY),
```

Comment:
Do you think you can use junit Parameterized tests here instead. I saw you use it in the later pr.

Reply:
done